components/admin_console/plugin_management/plugin_management.tsx

```diff
@@ -452,13 +495,13 @@ export default class PluginManagement extends AdminSettings {
 
     handleUpload = () => {
         this.setState({lastMessage: null, serverError: null});
-        const element = this.refs.fileInput;
-        if (element.files.length > 0) {
+        const element = this.refs.fileInput as HTMLInputElement;
+        if (element.files && element.files.length > 0) {
```

Comment:
You can use optional chaining here:
```suggestion
        if (element.files?.length > 0) {
```

Reply:
@devinbinnie optional chaining is throwing type error.`TS2532: Object is possibly 'undefined'.`

Reply:
Odd, we fixed that...leave it as it I suppose. Thanks again :)